projects/kubernetes/e2e/Makefile

```diff
@@ -0,0 +1,29 @@
+ORG?=docker
+IMAGE=kubernetes-for-desktop-e2e
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+
+kubernetes_version=v1.8.0
```

Comment:
Does this need to remain in sync with `kubernetes/Dockerfile`?

Reply:
Yes, sounds like it's better to keep the versions in sync. Right now this uses release binaries, but if one builds from source it'd be ideal if freshly built artefacts get used here too.